com.unity.uiwidgets/Runtime/engine2/UIWidgetsPanel.cs

```diff
@@ -94,30 +94,30 @@ public partial class UIWidgetsPanel : RawImage, IUIWidgetsWindow {
         protected void OnEnable() {
             base.OnEnable();
             D.assert(_wrapper == null);
+            _configurations = new Configurations();
             _wrapper = new UIWidgetsPanelWrapper();
             onEnable();
-            
+
             if (fonts != null && fonts.Length > 0) {
                 foreach (var font in fonts) {
-                    AddFont(font.family, font);
+                    AddFont(family: font.family, font: font);
                 }
             }
 
             _wrapper.Initiate(this, width: _currentWidth, height: _currentHeight, dpr: _currentDevicePixelRatio,
-                settings: _internalTextFonts);
-            _internalTextFonts.Clear();
+                _configurations: _configurations);
+            _configurations._internalTextFonts.Clear();
```

Comment:
please fix the similar issues in this file as I commented out before in UIWidgetsEditorPanel.cs